tests/scripts/gen_release_notes.sh

```diff
@@ -16,27 +16,27 @@ if [ -z "${GITHUB_USER}" ] || [ -z "${GITHUB_TOKEN}" ]; then
   help
 fi
 
-pr_list=$(git log --pretty="%s" --merges --left-only ${FROM_BRANCH}...${TO_TAG} | grep pull | awk '/Merge pull request/ {print $4}' | cut -c 2-)
+pr_list=$(git log --pretty="%s" --merges --left-only "${FROM_BRANCH}"..."${TO_TAG}" | grep pull | awk '/Merge pull request/ {print $4}' | cut -c 2-)
 
 # for releases notes
 function release_notes() {
   for pr in $pr_list; do
   # get PR title
-  backport_pr=$(curl -s -u ${GITHUB_USER}:${GITHUB_TOKEN} https://api.github.com/repos/rook/rook/pulls/${pr} | jq '.title')
+  backport_pr=$(curl -s -u "${GITHUB_USER}":"${GITHUB_TOKEN}" https://api.github.com/repos/rook/rook/pulls/"${pr}" | jq '.title')
   # with upstream/release-1.6 v1.6.8, it was giving extra PR numbers, so removing after PR for changing tag is merged.
-  if [[ "$backport_pr" =~ "build: Update build version to $TO_TAG for the release" ]]; then
+  if [[ "$backport_pr" =~ build:\ Update\ build\ version\ to\ $TO_TAG ]]; then
```

Comment:
It's really hard to read with the escape slashes. 

I would use globs for this
```suggestion
  if [[ "$backport_pr" == *"build: Update build version to $TO_TAG"* ]]; then
```